examples/wrap/index.html

```diff
@@ -30,7 +30,7 @@ <h1>Unwrapped</h1>
           <textarea id="unwrapped" placeholder="Enter text here..."></textarea>
         </div>
         <div>
-          <h1>Wrapped to 80 Characters</h1>
+          <h1>Wrapped to <input id="n" type="number" value="80" min="1"/> Characters</h1>
```

Comment:
what about 'width' or 'wrappedLen' or smthg? `n` is too generic.

Reply:
Done.